tox.ini

```diff
@@ -28,7 +28,7 @@ deps=
     flake8,flakeplus,pydocstyle: -r{toxinidir}/requirements/pkgutils.txt
 
 commands = pip install -U -r{toxinidir}/requirements/dev.txt
-           pytest -rxs -xv --cov=kombu --cov-report=xml --no-cov-on-fail {posargs}
+           python -bb -m pytest -rxs -xv --cov=kombu --cov-report=xml --no-cov-on-fail {posargs}
```

Comment:
So the `-bb` flag produces the bytes-str comparison warnings?

Reply:
`-b` produces warnings. `-bb` turns these warnings into errors. Without these flags the comparison "passes silently" but under Python 3 never returns `True`.